presto-hive/src/main/java/com/facebook/presto/hive/GenericHiveRecordCursor.java

```diff
@@ -58,7 +58,7 @@
 import static com.facebook.presto.spi.type.BigintType.BIGINT;
```

Comment:
Please fix your commit message (follow this: https://chris.beams.io/posts/git-commit/) lol ;)